floor_generator/lib/value_object/dao.dart

```diff
@@ -25,6 +27,7 @@ class Dao {
     this.deletionMethods,
     this.transactionMethods,
     this.streamEntities,
+    this.streamViews,
```

Comment:
⚠️Please update equals, hashCode and toString

Reply:
I'll also use `ListEquality` for this and the other `List`s instead of the current ` == ` comparison.